src/app/components/action-card-component/action-card-component.component.html

```diff
@@ -0,0 +1,107 @@
+<div class="container">
+  <div class="card">
+    <div class="card-top">
+      <h5 class="card-title" [ngStyle]="{ color: titleColor }">
+        {{ title }}
+      </h5>
+      <button>
+        <i class="bi bi-plus-circle"></i>
+        {{ addActivityText }}
+      </button>
+    </div>
+
+    <div class="card-body">
+      <div class="row" *ngFor="let row of cardContentRow">
+        <div class="accordion-header">
+          <div class="title-row">
+            <div class="icon" (click)="toggleList()">
+              <i class="bi bi-chevron-compact-up" *ngIf="isListOpen"></i>
+              <i class="bi bi-chevron-compact-down" *ngIf="!isListOpen"></i>
+            </div>
+            <div class="title">{{ row.title }}</div>
+            <span
+              *ngIf="!row.isDeletableAction"
+              class="badge bg-secondary d-inline-flex align-items-center"
+              >{{ row.badge }}</span
+            >
+          </div>
+
+          <div class="icons">
+            <i class="bi bi-pencil-square"></i>
+            <div class="delete-action" *ngIf="!row.isDeletableAction">
+              <span class="deleteText" (mouseover)="showPopover = true">{{
+                deleteText
+              }}</span>
+              <div *ngIf="showPopover" class="popover-container">
+                <div class="popover-text">
+                  <div class="popover-body">
+                    <strong>{{ row.popoverHeader }}</strong
+                    ><i (click)="closePopover()" class="bi bi-x"></i>
+                  </div>
+                  <div class="popover-body">
+                    {{ row.popoverBody }}
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="delete-action" *ngIf="row.isDeletableAction">
+              <i class="bi bi-trash"></i>
+            </div>
+          </div>
+        </div>
+
+        <div class="list-items" *ngIf="isListOpen">
+          
+            <ul *ngFor="let listItem of row.listItems">
+              <li>{{ listItem }}</li>
+            </ul>
+          
+        </div>
+        <hr />
+      </div>
+
+      <!-- <div class="row">
```

Comment:
if you leave a commented out code in the PR, this should be code with valuable information. If it is just a piece of code, that is not needed any more, remove it before making PR